cmd/config/init/flags.go

```diff
@@ -21,6 +22,12 @@ func addFlags(cmd *cobra.Command) {
 		if _, ok := Hubs[hubID]; !ok {
 			return fmt.Errorf("invalid hub ID: %s. %s", hubID, getAvailableHubsMessage())
 		}
+		rollappID := args[0]
+		if !validateRollAppID(rollappID) {
+			return fmt.Errorf("the provided RollApp ID '%s' is invalid. Please ensure it follows the format:"+
+				" 'letters_numbers_numbers',"+
```

Comment:
maybe change the message to be more like evmos and instead of `letters_numbers_numbers` make it `identifier_EIP155-version` and also give an example. 

> Evmos network follows the format of `identifier_EIP155-version` format.